Client/src/Store/user.ts

```diff
@@ -0,0 +1,5 @@
+import { writable } from 'svelte/store';
+
+const user = writable('');
```

Comment:
type it (maybe - see docs)

Reply:
type it? I didn't understand
But yeah I think it's a good use of stores, we just to be careful with the token expiary thingi